requirements.txt

```diff
@@ -5,5 +5,5 @@ PyYAML==3.11
 rasterio==0.30.0
 pyproj==1.9.4
 shapely==1.5.13
-git+https://github.com/GFDRR/thinkhazard_common.git#egg=thinkhazard_common
+git+https://github.com/GFDRR/thinkhazard_common.git@fbbadb3f50722541c0cc701bd20c31c6c3bb7d7c#egg=thinkhazard_common
```

Comment:
won't be required since https://github.com/GFDRR/thinkhazard_common/pull/10 has been merged